src/tests/ftest/server/metadata.py

```diff
@@ -399,11 +395,8 @@ def test_metadata_server_restart(self):
                 self.ior_managers[-1].verbose = False
 
                 # Add a thread for these IOR arguments
-                thread_manager.add(
-                    manager=self.ior_managers[-1], uuids=list_of_uuid_lists[index])
-                self.log.info(
-                    "Created %s thread %s with container uuids %s", operation,
-                    index, list_of_uuid_lists[index])
+                thread_manager.add(manager=self.ior_managers[-1], num_run=files_per_thread)
```

Comment:
This is incorrect, it should be:
```suggestion
                container_qty = files_per_thread * total_ior_threads
                thread_manager.add(manager=self.ior_managers[-1], num_run=container_qty)
```

Reply:
How? The previous behavior was:
```
list_of_uuid_lists = [
    [str(uuid.uuid4()) for _ in range(files_per_thread)]
    for _ in range(total_ior_threads)]
...
thread_manager.add(
    manager=self.ior_managers[-1], uuids=list_of_uuid_lists[index])
```
So each thread was given `files_per_thread` UUIDs, which is why `num_run=files_per_thread`

Reply:
Ah, right ignore me.